### ISSUE TITLE
KeyError: '1'

### ISSUE THREAD

**Issue by maojn** [bioinform/breakseq2#18]
Hi,
One of our users received the following error while running breakseq2:

Command-line: 
/usr/local/apps/breakseq2/2.2/bin/run_breakseq2.py --reference /fdb/bwa/indexes/hg19.fa --bams F82_001sorted.mdup.recal.bam --work breakseq2 --bwa bwa --samtools samtools --bplib_gff breakseq2_bplib_20150129.gff --nthreads 10 --sample F82_001
    INFO 2018-10-11 15:15:25,718 breakseq2_workflow   Generating breakpoint-library using breakseq2_bplib_20150129.gff
    Traceback (most recent call last):
      File "/usr/local/apps/breakseq2/2.2/bin/run_breakseq2.py", line 28, in <module>
        args.keep_temp, args.window, args.junction_length))
      File "/usr/local/Anaconda/envs_app/breakseq2/2.2/lib/python2.7/site-packages/breakseq2/breakseq_top.py", line 96, in breakseq2_workflow
        breakseq_index.generate_bplib(bplib_gff, reference, bplib, junction_length)
      File "/usr/local/Anaconda/envs_app/breakseq2/2.2/lib/python2.7/site-packages/breakseq2/breakseq_index.py", line 47, in generate_bplib
        flanks = sv.get_flanks()
      File "/usr/local/Anaconda/envs_app/breakseq2/2.2/lib/python2.7/site-packages/breakseq2/biopy/io/SV.py", line 120, in get_flanks
        seqA=seqs.get_sequence(*seqs.get_window(self.name, self.start))
      File "/usr/local/Anaconda/envs_app/breakseq2/2.2/lib/python2.7/site-packages/breakseq2/biopy/io/Fasta.py", line 20, in get_sequence
        seq = self.seqs[name].seq
    KeyError: '1'
    
I am not sure how to fix this error. Is there example files for me to test if the installation is good? Thank you.

Jean

**Comment by Ellis-Anderson** [comment]
Hey @maojn,

I got the same error, but came to realize it wasn't an issue with the program, but rather the gff files. Your reference, as well as mine, use 'chr1, chr2, chr3, ...' for the naming of chromosomes whereas the gff file uses '1, 2, 3, ...'

I used the following bash commands to change the chromosome naming scheme and then was able to run the software without issue. 

```
$ cat breakseq2_bplib_20150129.gff | awk -v OFS='\t' '$1="chr"$1' > breakseq2_bplib_20150129_chr.gff
$ sed -E 's/[0-9,X,Y].*/chr&/' breakseq2_bplib_20150129.ins > breakseq2_bplib_20150129_chr.ins
$ sed -E 's/[0-9,X,Y].*/chr&/' breakseq2_bplib_20150129.fna > breakseq2_bplib_20150129_chr.fna
```


This may not be the best answer, however, as this likely means that the gff files were made using a different reference/coordinates, and they're usually not perfectly interchangeable.
